Source/Engines/ConfigFilters.cs

```diff
@@ -0,0 +1,137 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace RealFuels
+{
+    [KSPAddon(KSPAddon.Startup.AllGameScenes, false)]
+    public class ModuleShowInfoUpdater : MonoBehaviour
+    {
+        protected bool run = true;
+        private void Update()
+        {
+            if (run)
+            {
+                Debug.Log("[RealFuelsFilters] Updated Info boxes");
+                foreach (AvailablePart ap in PartLoader.LoadedPartsList)
+                {
+                    if (ap.partPrefab.FindModuleImplementing<ModuleEngineConfigs>() is ModuleEngineConfigs mec)
```

Comment:
Edge case, but a part can have multiple MECs.

Reply:
Should be fixed, but I have no part to test on